packages/frontity-org-theme/src/processors/homepage/need-inspiration.tsx

```diff
@@ -0,0 +1,42 @@
+import { Processor } from "@frontity/html2react/types";
+import { css } from "frontity";
+import React from "react";
+
+import FrontityOrg from "../../../types";
+
+export const needInspirationSection: Processor<
+  React.HTMLProps<HTMLElement>,
+  FrontityOrg
+> = {
+  name: "need-inspiration-section",
+  test: ({ node }) =>
+    node.type === "element" &&
+    node.props?.className?.split(" ").includes("need-inspiration-section"),
+  processor: ({ node }) => {
+    if (node.type !== "element") {
+      return node;
+    }
+
+    //Style the section
+    node.props.css = css`
+      ${node.props.css};
```

Comment:
I would use here `overflow: hidden;` to avoid the gallery to overflow in the mobile view.

Reply:
I thought I had already made that :smile: